src/components/RichTextEditor/RichTextEditor.cy.tsx

```diff
@@ -148,309 +156,322 @@ const RichTextEditorWithValueSetOutside = ({ value }: { value: string }) => {
+        });
 
-    it('should render a raw html content state', () => {
-        cy.mount(<RichTextEditor value={'<b>this is bold</b> and <i>this italic</i>'} />);
+        it('should render a raw html content state', () => {
+            cy.mount(<RichTextEditor value={'<b>this is bold</b> and <i>this italic</i>'} />);
 
-        cy.get(RICH_TEXT_EDITOR).should('contain.text', 'this is bold and this italic');
-        cy.get('[contenteditable=true]').should('include.html', 'tw-font-bold');
-        cy.get('[contenteditable=true]').should('include.html', 'tw-italic');
-    });
+            cy.get(RICH_TEXT_EDITOR).should('contain.text', 'this is bold and this italic');
+            cy.get('[contenteditable=true]').should('include.html', 'tw-font-bold');
+            cy.get('[contenteditable=true]').should('include.html', 'tw-italic');
+        });
 
-    it('should render a plain text string content state', () => {
-        const TEXT = 'This is text';
-        cy.mount(<RichTextEditor value={TEXT} />);
+        it('should render a plain text string content state', () => {
+            const TEXT = 'This is text';
+            cy.mount(<RichTextEditor value={TEXT} />);
 
-        cy.get(RICH_TEXT_EDITOR).should('contain.text', TEXT);
-    });
+            cy.get(RICH_TEXT_EDITOR).should('contain.text', TEXT);
+        });
 
-    it('wraps the Editor in the component ', () => {
-        const TEXT = 'This is new text';
+        it('wraps the Editor in the component ', () => {
+            const TEXT = 'This is new text';
 
-        cy.mount(<RichTextEditorWithValueSetOutside value={TEXT} />);
-        cy.get(RICH_TEXT_EDITOR).should('contain.text', TEXT);
+            cy.mount(<RichTextEditorWithValueSetOutside value={TEXT} />);
+            cy.get(RICH_TEXT_EDITOR).should('contain.text', TEXT);
+        });
     });
 
-    it('should be editable by default ', () => {
-        cy.mount(<RichTextEditor />);
+    describe('Editable', () => {
+        it('should be editable by default ', () => {
+            cy.mount(<RichTextEditor />);
 
-        cy.get('[contenteditable=true]').should('exist');
-    });
+            cy.get('[contenteditable=true]').should('exist');
+        });
 
-    it('should not the able to edit when readonly ', () => {
-        cy.mount(<RichTextEditor readonly />);
+        it('should not the able to edit when readonly ', () => {
+            cy.mount(<RichTextEditor readonly />);
 
-        cy.get('[contenteditable=false]').should('exist');
-    });
+            cy.get('[contenteditable=false]').should('exist');
+        });
 
-    it('should type and insert text', () => {
-        cy.mount(<RichTextEditor />);
+        it('should type and insert text', () => {
+            cy.mount(<RichTextEditor />);
 
-        cy.get('[contenteditable=true]').click().type('hello');
+            cy.get('[contenteditable=true]').click().type('hello');
+        });
     });
```

Comment:
Add a space before every nested describe to make it easier to read

Reply:
Updated